src/com/company/M2/Task44.java

```diff
@@ -0,0 +1,31 @@
+package com.company.M2;
+
+public class Task44 {
+    public static void main(String[] args) {
+        System.out.println("1)[null]: " + makeNameClean(null));
+        System.out.println("2)[empty name]: " + makeNameClean(""));
+        System.out.println("3)bigar: " + makeNameClean("bigar"));
+        System.out.println("4)   ytuki: " + makeNameClean("   ytuki"));
+    }
+
+    public static String makeNameClean(String name) {
+        if (name == null || name.isBlank()) {
+            return "";
+        }
+        int amount = 0;
```

Comment:
name variable looks not matching enough. `amount` is more about of sum of numeric values.

Reply:
fixed